chsdi/templates/htmlpopup/bafu_klima_co2_ausstoss_gebaeude.mako

```diff
@@ -0,0 +1,48 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <%
+        lang = lang if lang in ('fr','it') else 'de'
+        hinweis_text = 'hinweis_%s' %lang
+        genh1_text = 'genh1_%s' %lang
+        gwaersceh1_text = 'gwaersceh1_%s' %lang
+        linkpdf_text = 'linkpdf_%s' %lang
+        linkbafu_text = 'linkbafu_%s' %lang
+    %>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.egid')}</td>
+        <td>${c['attributes']['egid'] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.hinweis')}</td>
+        <td>${c['attributes'][hinweis_text] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.co2_range')}</td>
+        <td>${c['attributes']['co2_range'] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.genh1')}</td>
+        <td>${c['attributes'][genh1_text] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.gwaersceh1')}</td>
+        <td>${c['attributes'][gwaersceh1_text] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.gexpdat')}</td>
+        <td>${c['attributes']['gexpdat'] or '-'}</td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.linkco2')}</td>
+        <td><a target="_blank" href="${c['attributes']['linkco2']}">${_('link')}</a></td>
+    </tr>
+    <tr>
+        <td class="cell-left">${_('ch.bafu.klima-co2_ausstoss_gebaeude.linkpdf')}</td>
+        <td><a target="_blank" href="${c['attributes'][linkpdf_text]}">${_('link')}</a></td>
```

Comment:
otherwise something like that would do it: https://github.com/geoadmin/mf-chsdi3/blob/eeccfe7c40cfdf376f045b07881ed1292a0b8be0/chsdi/templates/htmlpopup/gemeinde.mako#L17